languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -772,6 +785,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
                 <example>Qualquer um que queira vir será bem-vindo.</example>
                 <example>Até essa altura chegar decorrerão inúmeros conflitos.</example>
+
+                <!-- p5491 -->
+                <example>Vocês virem foi um erro.</example>
+                <!-- p5490: errors are intentional, do not fix -->
+                <example>Ele virá está noite?</example>
+                <example>Quero está.</example>
+                <example correction="Deixe estar"><marker>Deixe está</marker>.</example>
```

Comment:
If we are removing matches with `está`, we still need a separate rule/suggestion for it. The current correction we have may be wrong, but it points out an oddness about it and might still lead users to manually correct the sentence or notice what they wrote is wrong. I can add this to the issue's DoD, if you'd like.

Reply:
In any case, we should have it documented somewhere 🙂

Reply:
I'll just keep adding stuff to the confusion pairs, but I have some ideas about what we can do with diacritic-only confusion pairs based on past experience working with Spanish.

Reply:
Fixed!